xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -289,12 +289,40 @@ def _skip_target(
         label_str = bazel_labels.normalize_label(info.xcode_target.label)
         package_label_str = label_str.split(":")[0]
 
+        # Important notes:
+        #
+        # 1. This relies on implementation details of
+        # https://github.com/bazelbuild/rules_apple/blob/master/apple/internal/testing/apple_test_assembler.bzl#L100-L125
```

Comment:
I know we want columns of size `80` but at the same time I think it's important to link to the reference code path?

Reply:
Correct, links don't need to wrap, they just need to start on their own line if they would wrap.